core/src/main/kotlin/com/redmadrobot/debug/core/internal/DebugEvent.kt

```diff
@@ -0,0 +1,3 @@
+package com.redmadrobot.debug.core.internal
+
+public interface DebugEvent
```

Comment:
I wonder if this interface should be marked with `@DebugPanelInternal` since it is located inside internal package